include/aspect/material_model/steinberger.h

```diff
@@ -250,11 +213,21 @@ namespace aspect
 
 
       private:
-        bool has_background;
-        unsigned int first_composition_index;
+        /**
+         * The thermodynamic lookup equation of state.
+         */
+        EquationOfState::ThermodynamicTableLookup<dim> equation_of_state;
 
-        bool interpolation;
+        /**
+         * Boolean describing whether to calculate material properties
+         * including latent heat effects.
+         */
         bool latent_heat;
+
+
+        /**
+         * Boolean describing whether to use the lateral average temperature.
+         */
```

Comment:
If a comment basically just says what you can already figure out from the variable name, it doesn't add any new information. So I would either make the comment more useful ("Boolean describing whether to use the lateral average temperature for computing the viscosity, as opposed to the temperature on the reference adiabat."), or in this case, since this is already documented in the declare parameters function, I might just leave this without additional documentation.